.jenkins/short-perf-test-cpu.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+export PATH=/opt/conda/bin:$PATH
+
+echo "Running CPU perf test for PyTorch..."
+export OMP_NUM_THREADS=4
+export MKL_NUM_THREADS=4
+
+cd test/
+
+# TODO: Move this into its own file
+cat >compare_with_baseline.py << EOL
+
+import sys
+import numpy
+from scipy import stats
+
+mean_values = {
+  "commit": "92aeca1279265d24493dc6ced7dde9a368faf048",
+
+  "test_cpu_speed_mini_sequence_labeler": "2.62557",
+  "test_cpu_speed_mnist": "18.79468",
+}
+
+sigma_values = {
+  "commit": "92aeca1279265d24493dc6ced7dde9a368faf048",
```

Comment:
Do we need to update this periodically?

Reply:
Yes. I think there are a bunch of things to make these scripts more robust, but for moving them around I wanted to keep the code identical to what it is in ossci-job-dsl

Reply:
This kind of information shouldn't really live in the main repo. It's very specific to the hardware used to benchmark, so it's not even reproducible

Reply:
It is reproducible, because these are pinned to specific EC2 instance types.

Let's track improvements to benchmark scripts in #4948